validator/project_validator.go

```diff
@@ -4,7 +4,6 @@ import (
 	"bytes"
 	"context"
 	"fmt"
-	"math"
```

Comment:
I may have missed it but I don't see the code for 

> The top-level batchtime doesn't have any effect right now, so it can be removed. If any remaining YAMLs have it defined, evergreen validate produces a warning.

Is there maybe a push missing?

Reply:
The YAML has [strict parsing enabled](https://github.com/evergreen-ci/evergreen/blob/23fb65a4e32d54bb998f5aaca6faf7e9dfd19bc1/util/yaml.go#L32) so it warns on keys that don't match any field in the parser project data model.